provision-ecommerce.sh

```diff
@@ -9,5 +9,7 @@ set -x
 
 # Configure ecommerce
 docker-compose exec -T ecommerce bash -e -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_or_update_site --site-id=1 --site-domain=localhost:18130 --partner-code=edX --partner-name="Open edX" --lms-url-root=http://edx.devstack.lms:18000 --lms-public-url-root=http://localhost:18000 --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --sso-client-id=ecommerce-sso-key --sso-client-secret=ecommerce-sso-secret --backend-service-client-id=ecommerce-backend-service-key --backend-service-client-secret=ecommerce-backend-service-secret --from-email staff@example.com --discovery_api_url=http://edx.devstack.discovery:18381/api/v1/ --enable-microfrontend-for-basket-page=1 --payment-microfrontend-url=http://localhost:1998'
+docker-compose exec -T ecommerce bash -e -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_or_update_site --site-id=2 --site-domain=ecommerce.pearson.localhost:18130 --partner-code=PX --partner-name="PearsonX" --lms-url-root=http://edx.devstack.lms:18000 --lms-public-url-root=http://lms.pearson.localhost:18000 --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --sso-client-id=ecommerce-sso-key --sso-client-secret=ecommerce-sso-secret --backend-service-client-id=ecommerce-backend-service-key --backend-service-client-secret=ecommerce-backend-service-secret --from-email staff@example.com --discovery_api_url=http://discovery.pearson.localhost:18381/api/v1/ --enable-microfrontend-for-basket-page=1 --payment-microfrontend-url=http://localhost:1998'
+docker-compose exec -T ecommerce bash -e -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_or_update_site --site-id=3 --site-domain=ecommerce.main.localhost:18130 --partner-code=main --partner-name="Base" --lms-url-root=http://edx.devstack.lms:18000 --lms-public-url-root=http://lms.main.localhost:18000 --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --sso-client-id=ecommerce-sso-key --sso-client-secret=ecommerce-sso-secret --backend-service-client-id=ecommerce-backend-service-key --backend-service-client-secret=ecommerce-backend-service-secret --from-email staff@example.com --discovery_api_url=http://discovery.main.localhost:18381/api/v1/ --enable-microfrontend-for-basket-page=1 --payment-microfrontend-url=http://localhost:1998'
```

Comment:
Once I signed in to `http://lms.pearson.localhost/`, then I wanted to go to ecommerce `http://ecommerce.pearson.localhost:18130/login`. However I got this error at login:

![image](https://user-images.githubusercontent.com/40271196/219049456-6760bf3a-28cc-41de-ac73-022b4721339d.png)


I am not completely sure this solves the issue I am facing. But I would say there is a lack of consistency in what is in site configuration for eComm sites since all sites have the same root url:

![image](https://user-images.githubusercontent.com/40271196/219049863-ebea0c2c-ffdf-4132-9cf8-2fc7177a70ac.png)

And based on the name, it should be the LMS url for the specific site. See following screenshot. 

![image](https://user-images.githubusercontent.com/40271196/219050045-fcead0a5-2fc6-4ab4-9858-3d42f37bd7bb.png)

```suggestion
docker-compose exec -T ecommerce bash -e -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_or_update_site --site-id=2 --site-domain=ecommerce.pearson.localhost:18130 --partner-code=PX --partner-name="PearsonX" --lms-url-root=http://lms.pearson.localhost:18000 --lms-public-url-root=http://lms.pearson.localhost:18000 --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --sso-client-id=ecommerce-sso-key --sso-client-secret=ecommerce-sso-secret --backend-service-client-id=ecommerce-backend-service-key --backend-service-client-secret=ecommerce-backend-service-secret --from-email staff@example.com --discovery_api_url=http://discovery.pearson.localhost:18381/api/v1/ --enable-microfrontend-for-basket-page=1 --payment-microfrontend-url=http://localhost:1998'
docker-compose exec -T ecommerce bash -e -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_or_update_site --site-id=3 --site-domain=ecommerce.main.localhost:18130 --partner-code=main --partner-name="Base" --lms-url-root=http://lms.main.localhost:18000 --lms-public-url-root=http://lms.main.localhost:18000 --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --sso-client-id=ecommerce-sso-key --sso-client-secret=ecommerce-sso-secret --backend-service-client-id=ecommerce-backend-service-key --backend-service-client-secret=ecommerce-backend-service-secret --from-email staff@example.com --discovery_api_url=http://discovery.main.localhost:18381/api/v1/ --enable-microfrontend-for-basket-page=1 --payment-microfrontend-url=http://localhost:1998'
```


**Note:** Probably the same is happening for Discovery. Could you please check it as well?. @alexjmpb